tools/oesign/main.c

```diff
@@ -64,6 +64,11 @@ static const char _usage_sign[] =
     "  -o, --output-file        [optional] file name (with path) where the\n"
     "                           signature of the enclave image will be\n"
     "                           written.\n"
+    "  -z, --zero-base         [optional] Enable/disable creation of enclave\n"
```

Comment:
Should these come from the config file instead of parameters to oesign?

Reply:
There is a 'CreateZeroBaseEnclave' option enabled on conf files. Should I disable command line input for the same?

Reply:
enable/disable

Reply:
CreateZeroBaseEnclave, StartAddress are optional config parameters available to be configured from the config file.